### DISCUSSION TITLE
Minecraft MMRPG ❤- 2022

### DISCUSSION THREAD

**Discussion by Igcorreia** [google/model-viewer#2623] [Show and tell]
Hi everyone, wanna share the ❤. Was working already on a 3d viewer with Three JS when I stumble upon this.
I have 2 sections on the site, one is published without AR and the other has AR activated so the team can use it to test.

![cropped-Untitled_Artwork-1-pad7lpa23ae275c2i0lzvygrsphy6oyysxgpuuqgdg](https://user-images.githubusercontent.com/1076409/127412260-e3d9d062-a4de-4145-a384-287778b0ddfb.png)

1. https://www.snejhammer.com/assets/ - Public
2. https://www.snejhammer.com/assets-2-0/ - Private

https://user-images.githubusercontent.com/1076409/127412005-f233e2cc-7561-4543-96b1-0803b24d53b6.mp4

Official Website: https://www.snejhammer.com

**Comment by elalish** [comment]
Cool, thanks for sharing!
